90 enable posts and publish an article (#91)
ecoacoustics/website: content/posts/2024-recap.md

---
title: "Open Ecoacoustics 2024 Recap"
date: 2025-02-04T12:18:00+10:00
image: 'images/phil-a2o-unleashed.jpg'
tags: 
- News

---

Even if it's February, we would like to take a look back and celebrate some key events that happened last year in 2024.
<!--more-->

## 2024 Recap: Key Events and Achievements 

Last year was eventful with the Ecoacoustics Symposium in Melbourne and the "Make Your Own Recogniser" workshop 2.0 at ESA.

{{% figure src="/images/phil-a2o-unleashed.jpg" caption="Philip Eichinski at A2O Unleashed workshop at Melbourne Ecoacoustics Symposium 2024" width="100%"%}}

Our team continued to enhance platform operations, refine the Open Ecoacoustics data pipeline, and incorporate more analyses into analysis containers.  

> ## In 2024, we achieved significant milestones: 

Reviewer: ```suggestion
## 2024 milestones

 In 2024, we achieved significant growth: 
```

- 1.8 million recordings uploaded 
- 132 years of data added
- 186 TB of data uploaded
- 6.7 million recordings downloaded 
- 196 new users 

> ## This means that 24% of all of our data was uploaded to the A2O and Ecosounds just last year!

Reviewer: ```suggestion
This means that **24% of all of our data was uploaded to the A2O and Ecosounds just last year**!
```

---
title: "Open Ecoacoustics 2024 Recap"
date: 2025-02-03T12:18:00+10:00
tags: 
- General Update
---

I hope your 2025 is off to a great start. We at Open Ecoacoustics are excited to keep the community more updated on our activities and progress. This year, we're committed to providing more regular updates and sharing our work through these posts and on social media. 

## Follow Us on BlueSky and LinkedIn 

> We are excited to announce the launch of our accounts on [BlueSky](https://bsky.app/profile/openecoacoustics.bsky.social) and [LinkedIn](https://www.linkedin.com/company/open-ecoacoustics/). Follow us to stay in the loop! 

Reviewer: this sounds like it should be in the 2025 post

## 2024 Recap: Key Events and Achievements 

Last year was eventful with the Ecoacoustics Symposium in Melbourne and the "Make Your Own Recogniser" workshop 2.0 at ESA.

{{% figure src="/images/phil-a2o-unleashed.jpg" caption="Philip Eichinski at A2O Unleashed workshop at Melbourne Ecoacoustics Symposium 2024" width="50%"%}}

Our team continued to enhance platform operations, refine the Open Ecoacoustics data pipeline, and incorporate more analyses into analysis containers.  

### In 2024, we achieved significant milestones: 

> -  1.8 million recordings uploaded 

Reviewer: ```suggestion
- 1.8 million recordings uploaded 
```

> - 132 years of data duration 

Reviewer: ```suggestion
- 132 years of data added
```

- 186 TB of data uploaded

> -  6.7 million recordings downloaded 

Reviewer: ```suggestion
- 6.7 million recordings downloaded 
```

> - 196 users 

Reviewer: ```suggestion
- 196 new users 
```

> **This means that 24% of all data uploaded to A2O and Ecosounds was ingested last year!**

Reviewer: ```suggestion
**This means that 24% of all of our data was uploaded to the A2O and Ecosounds just last year!**
```

> Thank you for all our partners, users and contributers being part of this journey so far.

Reviewer: ```suggestion
Thank you to all our partners, users, and contributors being part of this journey so far.
```
